apps/src/BriskMatch.cpp

```diff
@@ -0,0 +1,127 @@
+
+#include <opencv2/core.hpp>
+#include <opencv2/features2d.hpp>
+#include <opencv2/highgui.hpp>
+#include <opencv2/imgcodecs.hpp>
+
+#include <iostream>
+#include <vector>
+
+#define IMG_QUALITY 95
+
+int main(int argc, const char* argv[])
+{
+    // vectors for match type and brisk
+    std::vector<cv::String> typeDesc;
+    std::vector<cv::String> typeAlgoMatch;
+    // Vector for images being compared
+    std::vector<cv::String> fileName;
+
+    typeDesc.push_back("BRISK");
+    typeAlgoMatch.push_back("BruteForce");
+
+    cv::CommandLineParser parser(
+        argc, argv,
+        "{ @image1 | ../data/basketball1.jpg | }"
+        "{ @image2 | ../data/basketball2.jpg | }");
+
+    // names of images to be compared
+    fileName.push_back(parser.get<std::string>(0));
+    fileName.push_back(parser.get<std::string>(1));
```

Comment:
I'm doubtful that we need anything above this line. `filename[0]` and `filename[1]` later can just be replaced with `argv[0]` and `argv[1]` respectively. The only other thing still being used is the "BruteForce" bit, which you can deal with further on.